operator/main.go

```diff
@@ -167,3 +199,24 @@ func main() {
 	}
 
 }
+
+func newExporter() (trace.SpanExporter, error) {
+	return stdouttrace.New(
+		// Use human readable output.
+		stdouttrace.WithPrettyPrint(),
+		// Do not print timestamps for the demo.
+		stdouttrace.WithoutTimestamps(),
+	)
+}
+
+func newResource() *resource.Resource {
+	r, _ := resource.Merge(
+		resource.Default(),
+		resource.NewWithAttributes(
+			semconv.SchemaURL,
+			semconv.ServiceNameKey.String("keptn-lifecycle-operator"),
+			semconv.ServiceVersionKey.String("v0.1.0"),
```

Comment:
I think we should update this version automatically

Reply:
Yes, I would like to follow up with version and hash but I suggest doing that in a follow-up PR since it requires changes in pipelines etc :)

Reply:
Created https://github.com/keptn-sandbox/lifecycle-controller/issues/117